src/execute_files.py

```diff
@@ -32,7 +32,8 @@ def create_outname(outdir, infile):
 def execute(infile, outfile, extras):
   """Create the command-line for an execution."""
   runner = extras['runner']
-  basename = os.path.splitext(os.path.basename(runner))[0]
+  # Only strip Windows suffxes because wasm.opt has one
```

Comment:
I guess a better way to say it is "we are stripping only windows suffixes (not all suffixes) because"